### PR TITLE
Add react to project

### PR DESCRIPTION
**Pull request by adrianMRJ**
## What does this change?

Add react functionality to our project.

## Why are you changing that?

Because the project need the react model to code in front-end.

## How do you manually test this?

You can run the index view and it should deploy a greeting.

## Merge/Deploy Checklist
* [x]   It has been reviewed and accepted.

### CODE REVIEW

**Review by jclopezdev — COMMENTED** [review]
What do you need the pli file for?

File: app/controllers/static_controller.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class StaticController < ApplicationController
+    def index
+        
+    end
+end
```
[the remark this answers is not in the document]
Comment (adrianMRJ) [reply app/controllers/static_controller.rb:5]: Solved

File: Gemfile [hunk 2]
```diff
@@ -7,7 +7,7 @@ ruby '2.5.1'
 gem 'rails', '~> 6.1.4', '>= 6.1.4.1'
 # Use sqlite3 as the database for Active Record
 gem 'pg'
-# Use Puma as the app server
+# Use Puma as the app servers
```
[the remark this answers is not in the document]
Comment (adrianMRJ) [reply Gemfile:10]: It's not really necessary, solved

File: app/controllers/v1/things_controller.rb [hunk 3]
```diff
@@ -0,0 +1,10 @@
+class V1::ThingsController < ApplicationController
+    def index
+        render json: {:things => [
+            {
+                :name => 'some-thing',
+                :guid => '0415674d-a0e8-4748-af68-a5f53a05c453'
+            }
+        ] }.to_json
+    end
+end
```
[the remark this answers is not in the document]
Comment (YilKanda) [reply app/controllers/v1/things_controller.rb:10]: Yes, you can solve it manually.

**Review by YilKanda — CHANGES_REQUESTED** [review]

File: app/controllers/v1/things_controller.rb (hunk 3 above)
[the remark this answers is not in the document]
Comment (adrianMRJ) [reply app/controllers/v1/things_controller.rb:10]: Ok, it's solved

**Review by YilKanda — APPROVED** [review]
LGTM!

**Review by YilKanda — APPROVED** [review]
LGTM!

**Pull request merged**